CONTRIBUTING.md

```diff
@@ -23,10 +23,10 @@ All commands should be run using yarn. If you haven't switched to [yarn](https:/
 
 > If you are familiar with npm then using yarn should be a breeze. You can keep using npm if you'd prefer but you will miss out on the safety and security of yarn
 
-Microcosm must manage multiple projects. To do that, we use [Lerna](https://lernajs.io). Setup the project with:
+Microcosm must manage multiple projects. To do that, we use [Lerna](https://lernajs.io). After [installing Lerna globally](https://lernajs.io/#getting-started), setup the project with:
 
 ```bash
-yarn bootstrap
+lerna bootstrap
```

Comment:
Yikes! Thanks for sending this in. We shouldn't have to install Lerna globally though. 

I think what happened is I discovered that I didn't need Lerna for package installation. [yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/) are sufficient to install all of the dependencies and interlink packages (for example, `microcosm-dom` depends on `microcosm`. 

Do you mind re-writing this to say something like (?):

> The microcosm repo manages multiple projects using [yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/). Each package lives within a sub-folder of `./packages` directory. 
>
> To install dependencies for all projects, run `yarn install` from the repo root.

Reply:
Ah cool, I wasn't aware of Yarn Workspaces! I've just pushed a commit with the change.